docs/src/tutorials/mathprogbase.md

```diff
@@ -0,0 +1,96 @@
+# Transitioning from MathProgBase
+
+MathOptInterface is a replacement for [MathProgBase.jl](https://github.com/JuliaOpt/MathProgBase.jl).
+However, it is not a direct replacement.
+
+## Transitioning a solver interface
+
+Writing a solver interface in MathOptInterface is more work than the equivalent
+interface in MathProgBase.
+
+For more information, read [Implementing a solver interface](@ref).
+
+## Transitioning the high-level functions
+
+MathOptInterface doesn't provide replacements for the high-level interfaces in
+MathProgBase. We recommend you use JuMP instead.
```

Comment:
Maybe "We recommend you use JuMP instead as a modelling layer on top of MOI"? Otherwise, it sounds a bit like you're telling people to use completely different packages.